nautobot_ssot_aristacv/diffsync/cvutils.py

```diff
@@ -40,7 +40,7 @@ def connect():
         call_creds = grpc.access_token_call_credentials(response.json()["sessionId"])
     # Set up credentials for CVaaS using supplied token.
     else:
-        cvp_url = "www.arista.io:443"
+        cvp_url = PLUGIN_SETTINGS["cvaas_url"]
```

Comment:
Should we use `.get` here and keep the original URL as default?

Reply:
Good idea! Updated.